### ISSUE TITLE
Second server on second co-located domain always fails on first start

### ISSUE THREAD

**Issue by jasonp18** [biemond/biemond-orawls#36]
Hi Edwin,

We have two SOA Suite domains (e.g. soa1domain, soa2domain) co-located on two machines . During each auto build, the only repeatable issue is that every time the second managed server (soa2_ms2) in the second domain (soa2domain) fails on first start. 

Machine 1: Admin(soa1), Admin(soa2), soa1_ms1, soa2_ms2
Machine 2: soa1_ms2, soa2_ms2

The current workaround is manually start at command line and enter credentials when prompted. The server will eventually stop automatically and request to restart via node manager (requirement for consensus migration basis). Restart Node Manager and then start the managed server from the Weblogic console.

<18-Jun-2014 05:56:07 o'clock BST> <Critical> <Security> <BEA-090518> <Could not decrypt the username attribute value of {AES}246rovWvZxqSyhCS4XAWfoREUsX/DwWJO77hHfM8TMg= from the file /
u01/app/oracle/admin/domains/ipt_cdp_domain/servers/cdp_server2/data/nodemanager/boot.properties. If you have copied an encrypted attribute from boot.properties from another domain into
/u01/app/oracle/admin/domains/ipt_cdp_domain/servers/cdp_server2/data/nodemanager/boot.properties, change the encrypted attribute to its cleartext value then reboot the server. The attri
bute will be re-encrypted. Otherwise, change all encrypted attributes to their cleartext values, then reboot the server. All encryptable attributes will be re-encrypted. The decryption f
ailed with the exception weblogic.security.internal.encryption.EncryptionServiceException.>

Please advise. Many thanks, Jason

**Comment by guptasachin1112** [comment]
HI Edwin,

 I have a scenario where I have installed Admin + managed_1 on machine A , and managed_2 on machine B. I am able to start the admin server and managed_1 on A machine using weblogic admin console and through command prompt. I had created a domain and then pack domain on machine A and then unpack domain on machine B. 

I have started the nodeManager on machine B and my node manger is also reachable through weblogic console. I have also enrolled my nodemanager on machine B using wlst. But while starting the second managed server on machine B , it failed every time.   it gives error as 
Error: Diagnostics data was not saved to the credential store.
Error: Validate operation has failed.
Need to do the security configuration first!
<Aug 27, 2015 7:41:56 PM> <FINEST> <NodeManager> <Waiting for the process to die: 15567>
<Aug 27, 2015 7:41:56 PM> <INFO> <NodeManager> <Server failed during startup so will not be restarted>
<Aug 27, 2015 7:41:56 PM> <FINEST> <NodeManager> <runMonitor returned, setting finished=true and notifying waiters>
^C

then I tried to start the second manged server from machine A (where Admin ) is installed. using startManaged command.  the second managed server on machine B is started but at last failed with the error message as 

<Aug 27, 2015 7:04:51 PM GMT+00:00> <Emergency> <Security> <BEA-090087> <Server failed to bind to the configured Admin port. The port may already be used by another process.>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Critical> <WebLogicServer> <BEA-000362> <Server failed. Reason: Server failed to bind to any usable port. See preceeding log message for details.>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to FAILED>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Error> <WebLogicServer> <BEA-000383> <A critical service failed. The server will shut itself down>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Error> <Server> <BEA-002606> <Unable to create a server socket for listening on channel "Default". The address 10.152.173.xx  might be incorrect or another process is using port 14100: java.net.BindException: Cannot assign requested address.>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to FORCE_SHUTTING_DOWN>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Notice> <Cluster> <BEA-000163> <Stopping "async" replication service>

Is there any mistake done during unpacking and starting nodeManager on machine B??

**Comment by guptasachin1112** [comment]
Error: Diagnostics data was not saved to the credential store.
Error: Validate operation has failed.
Need to do the security configuration first!
<Aug 27, 2015 7:41:56 PM> <FINEST> <NodeManager> <Waiting for the process to die: 15567>
<Aug 27, 2015 7:41:56 PM> <INFO> <NodeManager> <Server failed during startup so will not be restarted>
<Aug 27, 2015 7:41:56 PM> <FINEST> <NodeManager> <runMonitor returned, setting finished=true and notifying waiters>

Error messages from machine A
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Emergency> <Security> <BEA-090087> <Server failed to bind to the configured Admin port. The port may already be used by another process.>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Critical> <WebLogicServer> <BEA-000362> <Server failed. Reason: Server failed to bind to any usable port. See preceeding log message for details.>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to FAILED>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Error> <WebLogicServer> <BEA-000383> <A critical service failed. The server will shut itself down>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Error> <Server> <BEA-002606> <Unable to create a server socket for listening on channel "Default". The address 10.152.173.xx  might be incorrect or another process is using port 14100: java.net.BindException: Cannot assign requested address.>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to FORCE_SHUTTING_DOWN>
<Aug 27, 2015 7:04:51 PM GMT+00:00> <Notice> <Cluster> <BEA-000163> <Stopping "async" replication service>

**Comment by guptasachin1112** [comment]
FINEST NodeManager Waiting for the process to die: 15567
INFO NodeManager Server failed during startup so will not be restarted
NodeManager runMonitor returned, setting finished=true and notifying waiters

**Comment by guptasachin1112** [comment]
HI Edwin,

 I have a scenario where I have installed Admin + managed_1 on machine A , and managed_2 on machine B. I am able to start the admin server and managed_1 on A machine using weblogic admin console and through command prompt. I had created a domain and then pack domain on machine A and then unpack domain on machine B. 

I have started the nodeManager on machine B and my node manger is also reachable through weblogic console. I have also enrolled my nodemanager on machine B using wlst. But while starting the second managed server on machine B , it failed every time.   it gives error as 
Error: Diagnostics data was not saved to the credential store.
Error: Validate operation has failed.
Need to do the security configuration first!
FINEST NodeManager Waiting for the process to die: 15567
INFO NodeManager Server failed during startup so will not be restarted
NodeManager runMonitor returned, setting finished=true and notifying waiters


then I tried to start the second manged server from machine A (where Admin ) is installed. using startManaged command.  the second managed server on machine B is started but at last failed with the error message as 


Emergency Security BEA-090087 Server failed to bind to the configured Admin port. The port may already be used by another process.
Critical WebLogicServer BEA-000362 Server failed. Reason: Server failed to bind to any usable port. See preceeding log message for details.
Notice WebLogicServer BEA-000365 Server state changed to FAILED
Error WebLogicServer BEA-000383 A critical service failed. The server will shut itself down
Error Server BEA-002606 Unable to create a server socket for listening on channel "Default". The address 10.152.173.xx  might be incorrect or another process is using port 14100: java.net.BindException: Cannot assign requested address.
Notice WebLogicServer BEA-000365 Server state changed to FORCE_SHUTTING_DOWN
Notice Cluster BEA-000163 Stopping "async" replication service

Is there any mistake done during unpacking and starting nodeManager on machine B??
